Makefile

```diff
@@ -110,43 +121,26 @@ check-variables:
 		exit 1; \
 	fi
 
-engine-dependencies: check-variables check-sdk-scripts
-	@./fetch-engine.sh || (printf "Unable to continue without engine files\n"; exit 1)
-	@cd $(ENGINE_DIRECTORY) && make dependencies WIN32=$(WIN32)
-
 engine: check-variables check-sdk-scripts
 	@./fetch-engine.sh || (printf "Unable to continue without engine files\n"; exit 1)
-	@cd $(ENGINE_DIRECTORY) && make core WIN32=$(WIN32)
-
-utility: engine-dependencies engine
-	@test -f "$(ENGINE_DIRECTORY)/OpenRA.Utility.exe" || (printf "OpenRA.Utility.exe not found!\n"; exit 1)
+	@cd $(ENGINE_DIRECTORY) && make all
 
-core:
-	@command -v $(MSBUILD) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
+all: engine
+	@command -v $(MSBUILD) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.18."; exit 1)
```

Comment:
That is actually a lie. `msbuild` does not come with Mono. It is built and packaged separately. No distribution will ever ship it in the same package as Mono as `msbuild` contains non-free parts.

Reply:
This is referring to Mono (the project) rather than `mono` (the package).